app/grandchallenge/components/templates/components/civ_set_form.html

```diff
@@ -56,11 +57,26 @@ <h3>
               hx-get="{{ new_interface_url }}"
               hx-target="#new-interfaces"
               hx-swap="beforeend"
-      >Add value for new interface</button>
+              hx-indicator="#hx-add-interface-indicator"
+              hx-enable-me
+              disabled
+      >
+          Add value for new interface
+          <span id="hx-add-interface-indicator" class="htmx-indicator spinner-border spinner-grow-sm" role="status"></span>
+          <span class="spinner-grow spinner-grow-sm" role="status" hx-hide-me></span>

```

Comment:
The spinner grow doesn't communicate loading for me. I think it would be better if it was the actual spinner, that the content tells you "Loading...". Like it is on the bootstrap example page https://getbootstrap.com/docs/4.2/components/spinners/#buttons. The real button text then gets swapped in when it has loaded.

Reply:
I had dislike replacing the text completely because you then don't know what you are waiting for... each button has a placeholder.

Just played around a bit and the closest I got is:
![image](https://github.com/comic/grand-challenge.org/assets/7871310/e0c6d846-9e54-454b-97a3-a563ce134feb)

Which feels way too bulky. So I'll throw in the proverbial towel and go for a `Loading ...` + spinner setup.

Reply:
I think doing what is shown on the bootstrap page is fine.